packages/zeplin-css-extension/package.json

```diff
@@ -82,6 +82,12 @@
         "type": "switch",
         "id": "useRemUnit",
         "default": true
+      },
+      {
+        "name": "Use experimental padding, margin estimation",
+        "type": "switch",
+        "id": "useExperimentalLayout",
+        "default": true
```

Comment:
"spacing" can be confused with the spacing 👾 feature.

Reply:
That's right. It's not very creative but `useMarginPaddingEstimations` would be my choice. 😅  

Any thoughts?

Reply:
Hmm, one can use the estimation, but if we're going to make "experimental" an extra explanation, then I'd call this "Show/Display padding, margin estimations (experimental)". The id of the preference doesn't need to include experimental in it, since we can make it not experimental one day, so I'd just go with `showPaddingMargin` there.

berk might suggest us some better names here too.

Reply:
> Hmm, one can use the estimation, but if we're going to make "experimental" an extra explanation, then I'd call this "Show/Display padding, margin estimations (experimental)". The id of the preference doesn't need to include experimental in it, since we can make it not experimental one day, so I'd just go with `showPaddingMargin` there.
> 
> berk might suggest us some better names here too.

This sounds good to me! 👍 Let's go with what merih suggested:

```suggestion
        "name": "Show padding, margin estimations (experimental)",
        "type": "switch",
        "id": "showPaddingMargin",
        "default": true
```